### PR TITLE
Add missing README changes for multi-module

### PR DESCRIPTION
**Pull request by AbandonedCart**
This is just adding all of the other places the instructions change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,13 +46,13 @@ OR
...
 PS : for users who want to import the api into Eclipse IDE, you'll need to :
-  - build once : `` ./gradlew build ``
+  - build once : `` ./gradlew :library:build ``
   - Right click on the project
   - Select Build path > Configure Build Path > Source > Add Folder
-  - Select `build/generated/source/proto/main/java`
+  - Select `library/build/generated/source/proto/main/java`
   - Finish
 
 # Usage exemple (mostly how to login) :
```
Comment (Jari27) [inline README.md:58]: Please fix this typo while you're at it ;)

**exemple**

Comment (AbandonedCart) [inline README.md:58]: That is not the purpose of this PR. See 6d9471ff08eb6da29ce22555cda817ec29a20552

Comment (Jari27) [inline README.md:58]: I know. Didn't see the other PR though.

Comment (AbandonedCart) [inline README.md:58]: There is no PR. It was changed, but then changed back.

**Pull request merged**